circuits/cpp/barretenberg/cpp/src/barretenberg/common/fuzzer.hpp

```diff
@@ -184,6 +185,102 @@ concept InstructionWeightsEnabled = requires {
                                         typename T::InstructionWeights;
                                         T::InstructionWeights::_LIMIT;
                                     };
+
```

Comment:
This is a function I've added for the future. I hope to create a variadic template for buffer mutations of elements so that anyone can build a fuzzer in 5 minutes with cool mutations